go.sum

```diff
@@ -15,10 +15,12 @@ github.com/ElrondNetwork/elrond-vm v0.0.25 h1:FIALKEneTS6BsGgdKDUx16w81S1MvwYnLc
 github.com/ElrondNetwork/elrond-vm v0.0.25/go.mod h1:w0jMH4xA506twG5Gkki5MN54eQ0jPCg06XuHvCQg44g=
 github.com/ElrondNetwork/elrond-vm-common v0.0.0-20191203115206-691b00a6e76a/go.mod h1:ZakxPST/Wt8umnRtA9gobcy3Dw2bywxwkC54P5VhO9g=
 github.com/ElrondNetwork/elrond-vm-common v0.1.9/go.mod h1:ZakxPST/Wt8umnRtA9gobcy3Dw2bywxwkC54P5VhO9g=
-github.com/ElrondNetwork/elrond-vm-common v0.1.13 h1:Y5ZPA8aNkRzAeMBx78HeiRJikZoB8Vguf/+5fF6tE5g=
-github.com/ElrondNetwork/elrond-vm-common v0.1.13/go.mod h1:ZakxPST/Wt8umnRtA9gobcy3Dw2bywxwkC54P5VhO9g=
+github.com/ElrondNetwork/elrond-vm-common v0.1.17 h1:tfxM6YYbU7i44hxpPQmq5TebEJgauIVepx/qG2lDOVQ=
```

Comment:
go mod tidy

Reply:
Ran `go mod tidy` but `elrond-vm-common v0.1.17` still in `go.sum`.